lab2/src/task4/Main.java

```diff
@@ -0,0 +1,32 @@
+package task4;
+import java.util.*;
+
+class Book {
+    private String title;
+    private String author;
+    private int year;
+
+    public Book(String title, String author, int year) {
+        this.title = title;
+        this.author = author;
+        this.year = year;
+    }
+
+    @Override
+    public String toString() {
+        // TODO
+        return this.title + ", written by " + this.author + ", published in " + this.year;

```

Comment:
Try to split long lines of code, especially in toString (usually, you make a new line before +)